src/page/summary-step.page.ts

```diff
@@ -4,7 +4,7 @@ export class SummaryStepPage {
   private proceedToCheckout: ElementFinder;
 
   constructor () {
-    this.proceedToCheckout = $('.cart_navigation span');
+    this.proceedToCheckout = $('.cart_navigation .standard-checkout');
```

Comment:
try to find a locator that uses the title attribute

Reply:
Done